core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -325,6 +326,26 @@ class MasterSuite extends SparkFunSuite
     }
   }
 
+  test("SPARK-46205: Recovery with Kryo Serializer") {
+    val conf = new SparkConf(loadDefaults = false)
+    conf.set(RECOVERY_MODE, "FILESYSTEM")
+    conf.set(RECOVERY_SERIALIZER, "Kryo")
+    conf.set(RECOVERY_DIRECTORY, System.getProperty("java.io.tmpdir"))
+
+    var master: Master = null
+    try {
+      master = makeAliveMaster(conf)
+      val e = master.invokePrivate(_persistenceEngine()).asInstanceOf[FileSystemPersistenceEngine]
+      assert(e.serializer.isInstanceOf[KryoSerializer])
```

Comment:
```suggestion
      val persistenceEngine = master.invokePrivate(_persistenceEngine()).asInstanceOf[FileSystemPersistenceEngine]
      assert(persistenceEngine.serializer.isInstanceOf[KryoSerializer])
```

Reply:
Thank you but this was intention in order to avoid `Scalastyle` failure. Line 338 has 99 char already.

<img width="1528" alt="Screenshot 2023-12-01 at 6 25 35 PM" src="https://github.com/apache/spark/assets/9700541/5dd77608-a9c2-434f-80f4-676cfac4176b">